optimizely/marketing-website
# Fix error reporting in concat banner and account for root directory in trim utility function

This PR addresses inproper variable naming for error reporting in our concat banner try/catch blocks. The error was also present in the the `trackLead` response parsing. I noticed this by improper reporting in GA in the screenshot below:

#### To Test
- to test locally you would have to copy and past the concat try/catch block into the `dev` concat banner and then intentionally throw an error in the JS bundle and set breakpoints in the inspector to ensure proper error parsing.
- to test the trimpath function got the console and `window.optly.mrkt.utils.trimTrailingSlash('/')`

![screen shot 2015-03-02 at 12 02 36 pm](https://cloud.githubusercontent.com/assets/4656726/6445736/0b6e4a5c-c0d4-11e4-985a-3f3705a55c70.png)
